tests/apollo/util/skvbc_history_tracker.py

```diff
@@ -1085,6 +1085,9 @@ async def tracked_prime_for_state_transfer(
             kv = [(key, val)]
             await self.write_and_track_known_kv(kv, client1)
 
+        await self.bft_network.wait_for_replicas_to_collect_stable_checkpoint(self.bft_network.get_live_replicas(),
```

Comment:
where is it used?

Reply:
This synchronisation point makes sure all live Replicas will collect a Stable Checkpoint. Without it the test test_st_when_fetcher_crashes in SkvbcPersistenceTest suite was consistently failing, because the late Replica (stale_node) catches up from the inactive Window, while enough Replicas collect a Stable Checkpoint in order for State Transfer to start.